packages/cln_plugin/lib/src/cln_plugin_base.dart

```diff
@@ -134,11 +136,13 @@ class Plugin implements CLNPlugin {
         if (messageSocket!.trim().isEmpty) {
           continue;
         }
+        var jsonRequest = Request.fromJson(jsonDecode(messageSocket));
 
         /// FIXME: read the json request
         try {
-          // uncomment this to make dart happy
-          /*var response =*/ await _call("INSERT THE NAME OF THE METHOD", {});
+          var response = await _call(jsonRequest.method, {});
+          File('/home/swapnil/clightning4j/p_log.txt')
+              .writeAsString(response.toString());
```

Comment:
```suggestion
              .writeAsString(response.toString());
        print(Response(result: response).toJson());
```

You should do something like that, note that we are ignoring any type of error